pkg/apis/monitoring/v1/prometheus_types.go

```diff
@@ -1313,6 +1313,11 @@ type QueueConfig struct {
 	// Retry upon receiving a 429 status code from the remote-write storage.
 	// This is experimental feature and might change in the future.
 	RetryOnRateLimit bool `json:"retryOnRateLimit,omitempty"`
+	// SampleAgeLimit is samples older than the limit will be dropped.
+	// It requires Prometheus >= v2.50.0.
+	//
+	// +optional
+	SampleAgeLimit string `json:"sampleAgeLimit,omitempty"`
```

Comment:
there's a built-in type for validating durations.
```suggestion
	SampleAgeLimit *Duration `json:"sampleAgeLimit,omitempty"`
```

Reply:
Thanks.  Change string to Duration done.